contracts/StakingRewardsV2.sol

```diff
@@ -370,16 +376,16 @@ contract StakingRewardsV2 is
         }
 
         uint256 rewardUSDC = rewardsUSDC[_account];
-        if (rewardUSDC > 0) {
+        if (rewardUSDC / PRECISION > 0) {
```

Comment:
given that `rewardUSDC / PRECISION > 0` will evaluate to `true` 99% of the time, It makes sense to just define:

```
uint rewardUSDC'  = rewardUSDC / PRECISION;
if(rewardUSDC') {...}
```

to save some gas and it is also cleaner

Reply:
Changed